src/test/java/com/google/api/codegen/testdata/python_doc_main_library.baseline

```diff
@@ -879,6 +888,97 @@ class LibraryServiceApi(object):
             index_map=index_map)
         self._update_book_index(request, options)
 
+    def stream_shelves(
+            self,
+            options=None):
+        """
+        Test server streaming
+        gRPC streaming methods don't have HTTP equivalent and need not have google.api.http option.
+
+        Example:
+          >>> from library.library_service_api import LibraryServiceApi
+          >>> from google.gax import CallOptions, INITIAL_PAGE
+          >>> api = LibraryServiceApi()
+          >>> for element in api.stream_shelves():
+          >>>   # process element
+          >>>   pass
+
+        Args:
+          options (:class:`google.gax.CallOptions`): Overrides the default
+            settings for this call, e.g, timeout, retries etc.
+
+        Returns:
+          An iterator which yields :class:`google.example.library.v1.library_pb2.StreamShelvesResponse` instances.
+
+        Raises:
+          :exc:`google.gax.errors.GaxError` if the RPC is aborted.
+        """
+        request = library_pb2.StreamShelvesRequest()
+        return self._stream_shelves(request, options)
+
+    def stream_books(
+            self,
+            name,
+            options=None):
+        """
+        Test server streaming, non-paged responses.
+        gRPC streaming methods don't have HTTP equivalent and need not have google.api.http option.
+
+        Example:
+          >>> from library.library_service_api import LibraryServiceApi
+          >>> api = LibraryServiceApi()
+          >>> name = ''
+          >>> for element in api.stream_books(name):
+          >>>   # process element
+          >>>   pass
+
+        Args:
+          name (string): The name of the shelf whose books we'd like to list.
+          options (:class:`google.gax.CallOptions`): Overrides the default
+            settings for this call, e.g, timeout, retries etc.
+
+        Returns:
+          An iterator which yields :class:`google.example.library.v1.library_pb2.Book` instances.
+
+        Raises:
+          :exc:`google.gax.errors.GaxError` if the RPC is aborted.
+        """
+        request = library_pb2.StreamBooksRequest(
+            name=name)
+        return self._stream_books(request, options)
+
+    def discuss_book(
+            self,
+            requests,
+            options=None):
+        """
+        Test bidi-streaming.
+        gRPC streaming methods don't have HTTP equivalent and need not have google.api.http option.
+
+        EXPERIMENTAL: This method interface might change in the future.
+
+        Example:
+          >>> from library.library_service_api import LibraryServiceApi
+          >>> from google.example.library.v1 import library_pb2
+          >>> api = LibraryServiceApi()
+          >>> request = tagger_pb2.DiscussBookRequest()
```

Comment:
Where does `tagger_pb2` come from? It is not imported.

IMO, better to do `request = [tagger_pb2.DiscussBookRequest()]` -- this shows that the request is itself an iterator.